BuildHelpers/Public/Set-ModuleTypes.ps1

```diff
@@ -0,0 +1,107 @@
+function Set-ModuleTypes {
+    <#
+    .SYNOPSIS
+        EXPIRIMENTAL: Set TypesToProcess
+        
+        [string]$TypesPath in a module manifest
+
+    .FUNCTIONALITY
+        CI/CD
+
+    .DESCRIPTION
+        EXPIRIMENTAL: Set TypesToProcess
+        
+        [string]$TypesPath in a module manifest
+
+    .PARAMETER Name
+        Name or path to module to inspect.  Defaults to ProjectPath\ProjectName via Get-BuildVariables
+
+    .PARAMETER TypesToProcess
+        Array of .ps1xml files
+
+    .PARAMETER TypesRelativePath
+        Path to the ps1xml files relatives to the root of the module (example: ".\Types")
+
+    .NOTES
+        Major thanks to Joel Bennett for the code behind working with the psd1
+            Source: https://github.com/PoshCode/Configuration
+
+    .EXAMPLE
+        Set-ModuleTypes -TypesRelativePath '.\Types'
+
+        Update module manifiest TypesToProcess parameters with all the .ps1xml present in the .\Types folder. 
+
+    .LINK
+        https://github.com/RamblingCookieMonster/BuildHelpers
+
+    .LINK
+        about_BuildHelpers
+    #>
+    [cmdletbinding()]
+    param(
+        [parameter(ValueFromPipeline = $True)]
+        [Alias('Path')]
+        [string]$Name,
+
+        [string[]]$TypesToProcess,
+
+        [string]$TypesRelativePath
+    )
+    Process
+    {
+        if(-not $Name)
+        {
+            $BuildDetails = Get-BuildVariables
+            $Name = Join-Path ($BuildDetails.ProjectPath) (Get-ProjectName)
+        }
+
+        $params = @{
+            Force = $True
+            Passthru = $True
+            Name = $Name
+        }
+
+        # Create a runspace
+        $PowerShell = [Powershell]::Create()
+
+        # Add scriptblock to the runspace
+        [void]$PowerShell.AddScript({
+            Param ($Force, $Passthru, $Name)
+            $module = Import-Module -Name $Name -PassThru:$Passthru -Force:$Force
+            $module | Where-Object Path -notin $module.Scripts
+
+        }).AddParameters($Params)
+
+        #Invoke the command
+        $Module = $PowerShell.Invoke()
+
+        if(-not $Module)
+        {
+            Throw "Could not find module '$Name'"
+        }
+
+        $Parent = $Module.ModuleBase
+        $File = "$($Module.Name).psd1"
+        $ModulePSD1Path = Join-Path $Parent $File
+        if(-not (Test-Path $ModulePSD1Path))
+        {
+            Throw "Could not find expected module manifest '$ModulePSD1Path'"
+        }
+
+        if(-not $TypesToProcess)
+        {
+            $TypesPath = Join-Path -Path $Parent -ChildPath $TypesRelativePath
+            $TypesList = Get-ChildItem -Path $TypesPath\*.ps1xml
```

Comment:
Might be nitpicking but I like to use `/` nowadays just to cover xplat stuff if it's ever needed/possible, if that makes sense?

Reply:
Sounds good . I just updated this in the latest push to the branch. I also took the liberty to udpate `Set-ModuleFormats` while I was at it.